README.md

```diff
@@ -33,109 +30,70 @@ To install TorchOk locally, run:
 ```bash
-python run_sagemaker.py --input_path fsx://fs-0f79df302dcbd29bd/z6duzbmv/tz_jpg --config configs/siloiz_pairwise_xbm_resnet50_512d.yml --instance_type ml.g4dn.xlarge
+train.py -cp examples/configs -cn classification_cifar10 trainer.accelerator='cpu'
 ```
-In case something isn't working inside the Sagemaker container you can debug your model locally. 
-Specify `local_gpu` instance type when starting the job:
+During the training you can access the training and validation logs by starting a local TensorBoard:
 ```bash
-python run_sagemaker.py --config configs/cifar10.yml --instance_type local_gpu --volume_size 5 --input_path file://../data/cifar10
-``` 
+tensorboard --logdir ~/.cache/torchok/logs/cifar10
+```
+### Export to ONNX
+TODO
+### Test ONNX model
+TODO
 
 ## Run tests
 ```bash
-docker exec -it torchok bash
-cd torchok
 python -m unittest discover -s tests/ -p "test_*.py"
 ```
+## To be added soon (TODO)
+Tasks
+- MOBY (unsupervised training)
+- DetectionTask
+- InstanceSegmentationTask
 
-## Differences in configs sagemaker vs local machine
-### 1. Path to data folder
-#### Sagemaker
-```yml
-data:
-  dataset_name: ExampleDataset
-  common_params:
-    data_folder: "${SM_CHANNEL_TRAINING}"
-```
-#### Local machine
-```yml
-data:
-  dataset_name: ExampleDataset
-  common_params:
-    data_folder: "/path/to/data"
-```
+Backbones
+- Swin-v2
+- HRNet
+- ViT
+- EfficientNet
+- MobileNetV3
 
-### 2. Path to artifacts dir
-#### Sagemaker
-```yml
-log_dir: '/opt/ml/checkpoints'
-```
+Segmentation models
+- HRNet neck + OCR head
+- U-Net neck
 
-#### Local machine
-```yml
-log_dir: '/tmp/logs'
-```
-### 3. Restore path
-`do_restore` is a special indicator which was designed to be used for SageMaker spot instances training. 
-With this indicator you can debug your model locally and be free to leave the `restore_path` pointing to some
-common directory like `/opt/ml/checkpoints`, where TorchOk will search the checkpoints for.
-#### Sagemaker
-```yml
-restore_path: '/opt/ml/checkpoints'
-do_restore: '${SM_USER_ENTRY_POINT}'
-```
+Detection models
+- YOLOR neck + head
+- DETR neck + head
 
-#### Local machine
-```yml
-restore_path: '/opt/ml/checkpoints'
-do_restore: '<true-if-continue-training>'
-```
+Datasets
+- Stanford Online Products
+- Cityscapes
+- COCO
 
-## Mlflow
-To have more convenient logs it is recommended to name your experiment as ```project_name-developer_name```, so that all your experiments related to this project will be under one tag in mlflow
-```yml
-experiment_name: &experiment_name test-yourname
-```
-State all the model parameters in ```mlflow.runName``` in logger params
-```yml
-logger:
-  logger: mlflow
-  experiment_name: *experiment_name
-  tags:
-    mlflow.runName: "simclr_swinb_512d"
-  save_dir: "s3://sagemaker-mlflow-main/mlruns"
-  secrets_manager:
-    region: "eu-west-1"
-    mlflow_secret: "acme/mlflow"
-```
+Losses
+- Pytorch Metric Learning losses
+- NT-ext (for unsupervised training)
+
+Metrics
```

Comment:
Torchmetrics already have it, if we need to say that we rewrite some metrics, i think we will use Torchmetrics metric.